[2024-05-18T19:04:22.030+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-18T19:04:22.048+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2022-10-23T01:00:00+00:00 [queued]>
[2024-05-18T19:04:22.056+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2022-10-23T01:00:00+00:00 [queued]>
[2024-05-18T19:04:22.057+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-18T19:04:22.068+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): hello_task> on 2022-10-23 01:00:00+00:00
[2024-05-18T19:04:22.075+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=465) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-18T19:04:22.076+0000] {standard_task_runner.py:63} INFO - Started process 467 to run task
[2024-05-18T19:04:22.076+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_loader', 'hello_task', 'scheduled__2022-10-23T01:00:00+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/data_loader.py', '--cfg-path', '/tmp/tmp9g55srka']
[2024-05-18T19:04:22.078+0000] {standard_task_runner.py:91} INFO - Job 89: Subtask hello_task
[2024-05-18T19:04:22.119+0000] {task_command.py:426} INFO - Running <TaskInstance: data_loader.hello_task scheduled__2022-10-23T01:00:00+00:00 [running]> on host b5cbc22da81e
[2024-05-18T19:04:22.185+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_loader' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2022-10-23T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-10-23T01:00:00+00:00'
[2024-05-18T19:04:22.187+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-18T19:04:22.188+0000] {logging_mixin.py:188} INFO - 2022-10-23/
[2024-05-18T19:04:22.189+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-18T19:04:22.190+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-18T19:04:22.197+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_loader, task_id=hello_task, run_id=scheduled__2022-10-23T01:00:00+00:00, execution_date=20221023T010000, start_date=20240518T190422, end_date=20240518T190422
[2024-05-18T19:04:22.210+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-18T19:04:22.224+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-18T19:04:22.226+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
